### ISSUE TITLE
Chemistry package (mail) sorting pipe actually goes to an operating room

### ISSUE THREAD

**Issue by GutsySS13** [Apollo-Community/ApolloStation#15]
The sorting pipe labled for chemistry doesn't actually go there, this means that Chemistry will have to collect small orders themselves, or antags can't post them bombs.

I suggest moving the sorting pipe so it directs pipes to chemistry as the current disposal chute just dumps into the main disposal network outbout only. It will take some minor pipe layout changes to fix, see linked document for my suggestion.

https://drive.google.com/file/d/0Bw3ZV7xxvq78N1JfNHU2a0FyLTA/view?usp=sharing
Basically swap the sorting pipes over (red line) so the chemistry sorting pipe is in the correct location, and link it streight to chemistry disposal directly by going up and left. The waste disposal in the corridor outside will need it's own seperate pipe to connect, but it's quite close to the main loop (streight down).

**Comment by Kwask** [comment]
Fixed in e17ed50e95
